build/Dockerfile.nginx

```diff
@@ -5,10 +5,12 @@ ARG NJS_DIR
 ARG NGINX_CONF_DIR
 ARG BUILD_AGENT
 
-RUN apk update && apk upgrade && apk add --no-cache libcap \
+RUN apk add --no-cache libcap \
     && mkdir -p /var/lib/nginx /usr/lib/nginx/modules \
     && setcap 'cap_net_bind_service=+ep' /usr/sbin/nginx \
     && setcap -v 'cap_net_bind_service=+ep' /usr/sbin/nginx \
+    # Update packages for CVE-2023-52425
```

Comment:
How did you catch this CVE? Was it showing up in a security scan?

Reply:
The build failed because of it, but it didn't upload to the security tab. The change to the workflow should fix that.